kolibri/plugins/coach/assets/src/modules/pluginModule.js

```diff
@@ -40,6 +40,11 @@ export default {
       state.toolbarRoute = toolbarRoute;
     },
   },
+  getters: {
+    multipleClasses(state) {
+      return state.classList.length > 1;
+    },
```

Comment:
* since this is a coach-wide getter, recommend renaming to something that conveys its intention a little more - for example `classListPageEnabled`
* also heads-up that the root handler could use a little clean-up (seems to call classroom resource twice?), and maybe even could reference this getter: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/coach/assets/src/routes/index.js#L18-L37

Reply:
Have updated both of these.